apps/condo/domains/user/schema/OidcClient.js

```diff
@@ -65,6 +65,17 @@ const OidcClient = new GQLListSchema('OidcClient', {
             type: 'Text',
         },
 
+        isEnabled: {
+            schemaDoc:
+                'A switch that allows you to disable some OIDC clients. ' +
+                'If an OIDC client is disabled, it cannot be used for OIDC authorization. ' +
+                'Used mainly by developers portal to create OIDC client before publishing an application, ' +
+                'and enable OIDC after publishing.',
+            type: 'Checkbox',
+            isRequired: true,
+            defaultValue: false,
```

Comment:
may be true?

Reply:
Doesn't really matter, since there's only 2 ways of creating it. Via dev-portal, which will always pass this parameter. Or via admin panel, where you need to specify it explicitly.